### DISCUSSION TITLE
Update npm version

### DISCUSSION THREAD

**Discussion by gponty** [vercel/community#6171] [Help]
### Summary

Hi, I need "npm":">=9.6.7", possible to upgrade version ?
ty

### Steps to Reproduce

Deploy sveltekit project with add-to-calendar-button dependencies.

**Comment by amyegan** [comment]
Hey @gponty. There are a few ways to set NPM version for a project. You can find them here: https://vercel.com/guides/how-do-i-use-the-latest-npm-version-for-my-vercel-deployment
